content/_includes/devices/iphone.svg

```diff
@@ -1,13 +1,12 @@
 <svg
 	data-device="iphone"
-  aria-labelledby="iphoneID"
   role="presentation"
   xmlns="http://www.w3.org/2000/svg"
   width="250px"
   height="500px"
 	viewBox="0 0 250 500"
 >
-<title id="iphoneID">iphone</title>
+<title>iphone</title>
```

Comment:
Since icons are repeated, we were causing duplicate-ID issues

Reply:
and I believe this title tag, when used in a link, is read via screen reader. So right now, on a post on the resrouce page, it reads that the item I am on is a Link, The SVG title= "link' for instance on the Link Posts, and the Span text, which in that example is "link". So, it says "Link link link" for each of the link icons.

Reply:
If possible, maybe the span text there should be view all posts tagged with "link" or something of the like.

Reply:
![Screen Shot 2020-02-18 at 4 21 37 PM](https://user-images.githubusercontent.com/1581694/74783259-ea5a0300-526a-11ea-919d-747567dd43a6.jpg)
![Screen Shot 2020-02-18 at 4 20 29 PM](https://user-images.githubusercontent.com/1581694/74783261-eaf29980-526a-11ea-9007-bfef8f03a003.jpg)

Reply:
![Screen Shot 2020-02-18 at 4 19 15 PM](https://user-images.githubusercontent.com/1581694/74783303-0362b400-526b-11ea-8b63-1b60e18ae499.jpg)

Reply:
Asked around a bit, and it sounds like screen readers are inconsistent with inline SVG, and the recommendation is to use `aria-hidden focusable="false"` on the icons, no need for `<title>`, and label them in HTML. I think your proposal for a bit more label context is also good.